hw_a.py

```diff
@@ -0,0 +1,63 @@
+"""
+TASK 1
+Write a class to represent a Cash Register.
+You class should keep the state of price total and purchased items
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+"""
+
+
+class CashRegister:
+    number_of_items = 0
+    total_price = 0
+
+    def __init__(self, item, price):
+
+        self.item = item
+        self.price = price
+
+
+        self.total_items = None # {'item': 'price'}
+        self.total_price = 0
+        self.discount = 0
+        CashRegister.number_of_items += 1
+        CashRegister.total_price += price
+
+
+    def add_item(self, item, price):
+        self.append(item,price)
+
+    def remove_item(self, item, price):
+        self.item.remove(item, price)
+
+    def apply_discount(discount):
+        if self.total_price > 10:
+            reduce = self.total_price - self.discount * self.total_price
+            return reduce
+        return False
```

Comment:
Instead of return False, it should either return the total price or the reduced price